### PR TITLE
Don't let people to submit if they are marked as submission blocked

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

We need to present blocked candidates (via the duplicate matcher) with an interstitial page informing them they are blocked and need to contact support for their application to be unlocked

## Changes proposed in this pull request

Screenshot 2024-01-19 at 09 41 35

## Guidance to review

To mark someone as blocked submission

1. Enter on a application form in support 
2. Click on the email of the candidate (breadcrumb)
3. Click "Block account"
4. Check "Account submission blocked"
5. Click "Continue"

Now try to submit with the candidate.

I will also advise to unblock and try to submit again.

## Link to Trello card

https://trello.com/c/eGHp0pBo/1201-apply-re-introduce-blocked-submission-landing-page

### CODE REVIEW

**Comment by tomas-stefano** [comment]
[Apply: Blocked candidates are able to submit applications (Re-introduce blocked submission landing page)](https://trello.com/c/eGHp0pBo/1201-apply-blocked-candidates-are-able-to-submit-applications-re-introduce-blocked-submission-landing-page)

File: app/views/candidate_interface/continuous_applications/course_choices/blocked_submissions/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1 class="govuk-heading-l">Submission blocked</h1>
+<p class='govuk-body'>You’ve created more than one account.</p>
+<p class='govuk-body'>You can no longer submit applications from this account. Visit your other account to continue your application.</p>
+<p class='govuk-body'>You can apply for up to <%= @max_course_choices %> courses at a time.</p>
+<p class='govuk-body'>Email becomingateacher@digital.education.gov.uk if you have any questions.</p>
```
Comment (inulty-dfe) [inline app/views/candidate_interface/continuous_applications/course_choices/blocked_submissions/show.html.erb:5]: Using " and ' in attribute values

**Review by inulty-dfe — APPROVED** [review]
Looks good, just a little inconsistency with quotations used in the template

File: app/views/candidate_interface/continuous_applications/course_choices/blocked_submissions/show.html.erb (hunk 1 above)
Comment (tomas-stefano) [reply app/views/candidate_interface/continuous_applications/course_choices/blocked_submissions/show.html.erb]: Done!

File: app/views/candidate_interface/continuous_applications/course_choices/blocked_submissions/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h1 class="govuk-heading-l">Submission blocked</h1>
```
Comment (vassyz) [inline app/views/candidate_interface/continuous_applications/course_choices/blocked_submissions/show.html.erb:1]: This content is used in multiple places. Can't we have it in one partial?

Comment (tomas-stefano) [reply app/views/candidate_interface/continuous_applications/course_choices/blocked_submissions/show.html.erb:1]: The other is the 2023 content that we don't show anymore and will be deleted in my cleanup PR part 2, so I think it is best to leave for deletion.

**Review by vassyz — APPROVED** [review]

**Pull request merged**